### DISCUSSION TITLE
How to play music from iPhone to AVM Fritz!Box Media-Renderer

### DISCUSSION THREAD

**Discussion by witzker** [UnicornsOnLSD/finamp#321] [General]
Hi,

I want to get rid of Synology Music station & IOS App

So I installed Jellyfin as Server & IOS App:

and Finamp on my iPhone

BUT now I want to play my music from iPhone to a AVM Fritz!Box Media-Renderer

in Audio station App, I have an icon to choose the Player.

![image](https://user-images.githubusercontent.com/4120266/187781085-6404fc33-771e-4f1d-af66-bd8484804ba0.png)

How do I achieve this in Jellyfin or Finamp APP ?

Hope for help

**Comment by Chaphasilor** [comment]
This requires DLNA cast functionality, which is not yet implemented in Finamp. You could try to use the "cast" button in the Jellyfin app and see if thaz lets you cast music to your media renderer.

**Comment by witzker** [comment]
THX
Bad News!
Jelyfin amp is buggy - I was very happy to find Finamp which for me is much better!!

Is this feature already in the pipeline?
Is there a workaround, as I need only sound e.g.
Is it generally possible to direct the entire audio output of the iPhone to the render with any setting or app?
Like it's possible with audio to a Bluetooth Speaker.
What do you Think?

**Reply by Chaphasilor** [reply]
Yeah, the Jellyfin app isn't great for music playback, but it should have support for DLNA casting. Does this work? Because if not, Finamp won't be able to do this either.

I believe @rom4nik wants to implement cast functionality at some point, but there is some additional work needed before this can be done.

I'm not sure if there is an iOS app for casting all audio via DLNA, you'll probably need an AirPlay-compatible receiver for this.

**Reply by UnicornsOnLSD** [reply]
Yeah AirPlay is probably your best bet, UPNP will likely be implemented alongside casting.
